src/Tribe/Main.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Main Tribe Loxi class.
+ */
+
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+if ( ! class_exists( 'Tribe__Loxi__Main' ) ) {
```

Comment:
I think returning early here will remove an indentation level or maybe to combine this one with the one above.

```php
if ( ! defined( 'ABSPATH' ) || class_exists( 'Tribe__Loxi__Main' ) )
```

Reply:
I feel like I've seen some weirdness with doing class_exists checks at the top and not wrapping. Maybe it was only for functions. I'll test it out.